qiskit_experiments/curve_analysis/curve_analysis.py

```diff
@@ -135,122 +148,115 @@ def models(self) -> List[lmfit.Model]:
     def _run_data_processing(
         self,
         raw_data: List[Dict],
-        models: List[lmfit.Model],
-    ) -> CurveData:
+    ) -> ScatterTable:
```

Comment:
Is it okey to drop argument?

Reply:
I'm okay with this since probably not many people were directly interacting with it.